cpu/efm32/efm32-features.mk

```diff
@@ -1,4 +1,4 @@
 # This file provides defaults for additional EFM32-specific features. You
 # should override them from the command line, or in your Makefile. Note that
 # some features may not be applicable to all EFM32 boards or CPUs.
-export EFM32_LEUART_ENABLED ?= 1
+export CONFIG_EFM32_FEATURE_LEUART ?= 1
```

Comment:
True, but that would break backwards compatibility. Since Kconfig is still optional, I need to have a default value for this option, right?

If I would remove it, support for LEUART would always be disabled.

Reply:
But the only use for this variable is to set a CFLAG when Kconfig is not being used. So you can just keep the old value for the Makefile variable, and only check for its value if Kconfig was not used:
```makefile
ifndef CONFIG_KCONFIG_CPU_EFM32
  ifeq (1,$(EFM32_FEATURE_LEUART))
    CFLAGS += -DCONFIG_EFM32_FEATURE_LEUART=1
  endif
endif
```